server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaBuilder.java

```diff
@@ -406,6 +426,15 @@ public CentralDogmaBuilder maxNumBytesPerMirror(long maxNumBytesPerMirror) {
         return this;
     }
 
+    /**
+     * Sets whether xDS control plane is enabled or not.
+     * If unspecified, xDS control plane is enabled.
+     */
+    public CentralDogmaBuilder xdsControlPlaneEnabled(boolean xdsControlPlaneEnabled) {
```

Comment:
Note) This PR starts leaking xDS logic into the core implementation.

Reply:
It would not make sense `xdsControlPlaneEnabled` is set to `true` when `XDS_CONTROL_PLANE_FOUND == false`

Reply:
Optional) Otherwise, we may generalize this to plug-in plugins. 
```
interface Plugin {
   // A unique name for this plugin
   String name();
}
```
```json5
{
  # All plugins are enabled by default.
  "plugins": {
    "include": [ "mirror" ]
    "exclude": [ "xds" ]
  }
}
```

Reply:
> xdsControlPlaneEnabled is set to true when XDS_CONTROL_PLANE_FOUND == false

So, I've added a logic for that:
https://github.com/line/centraldogma/pull/993/files#diff-170293b368ccad4c74d5f040277fa7439a76a671becd4db7dfbc59576bac95a0R301

> Otherwise, we may generalize this to plug-in plugins.

That's a good suggestion. Let me apply for that.

Reply:
@jrhee17, @ikhoon Changed to configure each plugin. PTAL. 🙇